tutor/src/screens/task/step/exercise-free-response.js

```diff
@@ -171,7 +181,7 @@ class FreeResponseInput extends React.Component {
             <AnswerButton
               size="lg"
               data-test-id="submit-answer-btn"
-              disabled={ux.isSubmitDisabled}
+              disabled={ux.isSubmitDisabled || ux.isOverWordLimit}
               onClick={this.onSave}>
```

Comment:
rest of pr looks great, but how about changing the `isSubmitDisabled` in ux to check the word count itself that way it's controlling the logic

Reply:
:+1: